package.json

```diff
@@ -33,7 +33,8 @@
   ],
   "scripts": {
     "test": "npm run lint && metatests test/",
-    "lint": "eslint --ignore-path .gitignore .",
+    "lint": "eslint . --ignore-path .gitignore",
```

Comment:
Is this needed?

Reply:
I found that in most of repositories the dot goes after eslint not at the end of a command. Alright, I'll change it back.